automation/e2e-upgrade-functests/run.sh

```diff
@@ -20,9 +20,30 @@ NAMESPACE=${1:-kubevirt}
 RELEASE_BRANCH=${CI_BRANCH}
 if [[ -z ${RELEASE_BRANCH} ]] || [[ ${RELEASE_BRANCH} == "main" ]]
 then
-  # Get the latest release branch
-  RELEASE_BRANCH=$(curl -s 'https://api.github.com/repos/kubevirt/ssp-operator/branches?per_page=100' |
-    jq -r '[.[].name | select(startswith("release-v"))] | max_by(ltrimstr("release-v") | split(".") | map(tonumber))')
+  LATEST_RELEASE_BRANCH=""
+  PAGE=1
+  BRANCHES=$(curl -s "https://api.github.com/repos/kubevirt/ssp-operator/branches?page=1")
```

Comment:
Leave the `per_page=100` parameter here. We want to do small number of requests.